aten/src/ATen/native/cpu/layer_norm_kernel.cpp

```diff
@@ -280,7 +280,7 @@ void LayerNormBackwardKernelImpl(
   AT_DISPATCH_FLOATING_TYPES(
       X.scalar_type(), "LayerNormBackwardKernelImpl", [&]() {
         LayerNormBackwardKernelImplInternal<scalar_t>(
-            dY, X, mean, rstd, gamma, M, N, dX, dgamma, dbeta);
+            dY.is_contiguous() ? dY : dY.contiguous(), X, mean, rstd, gamma, M, N, dX, dgamma, dbeta);
```

Comment:
If you are doing this check here now, can you please remove it from derivatives.yaml to avoid duplication?